tests/rules/test_cat_dir.py

```diff
@@ -1,15 +1,63 @@
+import gettext
+import os
+import warnings
+
 import pytest
+
 from thefuck.rules.cat_dir import match, get_new_command
 from thefuck.types import Command
 
+# patch this so the libc locale is available in the testing virtualenv
+gettext._default_localedir = '/usr/share/locale'
```

Comment:
This is dirty and I don't like it, but the default `_default_localedir` includes `sys.base_prefix` and `share/locale` isn't generally linked into a virtual environment.